packages/webapp/src/single/MediaView.tsx

```diff
@@ -203,7 +209,8 @@ export const MediaView = () => {
           </div>
           { showDetails &&
             <div className="md:flex-shrink-0 md:w-90">
-              <Details entry={current} dispatch={dispatch} />
+              <Details entry={current} dispatch={dispatch}/>
+              <HelpModal showHelp={showHelp} setShowHelp={setShowHelp} dispatch={dispatch}/>
```

Comment:
Would be better to pass there real mappings also instead of hardcoded strings.

Reply:
this one: `const hotkeysToAction = {`

Reply:
Good points. But I don't think it's necessary as it only have some important shortcuts.

Reply:
Maybe right now yes, but also there is introduced "plugins" so I think there will be extendable shortcuts list and hardcoded help will be useless.

Reply:
Have you read the Plugin documentation?

> Extending the web application is not yet supported. This feature is planned for the future.

So yeah, I don't think there's anything useless here, as shortcuts are a built-in feature in the web app.

Talking about plugin, to implement a plugin design pattern in React is feasible and can be approached by creating a wrapper component that encapsulates the plugin's logic and functionality. 

The wrapper acts as a higher-order component or context provider, managing the plugin’s lifecycle, configuration, and behavior. 

This approach allows you to inject plugins into the React component tree while maintaining separation of concerns.